atlasdb-client/src/main/java/com/palantir/atlasdb/schema/stream/StreamStoreDefinitionBuilder.java

```diff
@@ -45,7 +45,21 @@ public StreamStoreDefinitionBuilder(String shortName, String longName, ValueType
     }
 
     public StreamStoreDefinitionBuilder hashFirstRowComponent() {
```

Comment:
Should this be deprecated?

Reply:
Talked offline:

@amarzoca says: "in principle the old method should be deprecated, but that would mean that anyone using it would get deprecated warnings until they migrate all their data", which makes sense.

Reply:
Lets leave this not deprecated, until we have a migration story. Currently, our stand should be that this is supported for new stream stores only. 

^Worth adding in schema docs.